src/Migration/Action_Migrator.php

```diff
@@ -66,4 +69,8 @@ public function migrate( $source_action_id ) {
 			return $destination_action_id;
 		}
 	}
+
+	public function set_log_migrator( Log_Migrator $log_migrator ) {
```

Comment:
Rather than a public setter, I think the `Log_Migrator` should be injected through the constructor to match the way the source & destination stores are injected.

That will mean the order of the `Action_Migrator` and `Log_Migrator`'s construction will need to be reversed in `Migration_Runner`, but I don't _think_ that will cause any issues.

The other option is to trigger a hook before `$this->source->delete_action( $source_action_id )`, but I do like the approach of calling `$this->log_migrator->migrate()` directly.